release-gap-package

```diff
@@ -594,6 +595,26 @@ for EXT in $ARCHIVE_FORMATS ; do
 done
 
 
+######################################################################
+#
+# Upload PDF
+#
+cd "$TMP_DIR"
+echo ""
+
+FULLNAME="$TMP_DIR/$BASENAME/$PDFFile"
+if [ ! -f "$FULLNAME" ] ; then
+	error "could not find PDF"
+fi
+notice "Uploading PDF"
+response=$(curl --fail --progress-bar -o "$TMP_DIR/upload_pdf.log" \
+	-X POST "$UPLOAD_URL/$RELEASE_ID/assets?name=$BASENAME.pdf" \
+	-H "Accept: application/vnd.github.v3+json" \
+	-H "Authorization: token $TOKEN" \
+	-H "Content-Type: application/pdf" \
+	--data-binary @"$FULLNAME")
```

Comment:
It'd be nicer if that upload code was in a function which gets called with the filename and mime type (and the logname could just be `$TMP_DIR/${filename}.log`.

*Even nicer* would be to write all of this hellish shell code in e.g. Python :joy:

Needless to say, I don't expect you to do either of these, just had to get it off my chest (the current state is my fault, of course)